UPGRADE-2.0.md

```diff
@@ -0,0 +1,3 @@
+### UPGRADE FROM 1.X TO 2.0
+
+1. Resources configuration have been changed from yaml to php.
```

Comment:
You should describe here the difference in configuration for the end application